README.md

```diff
@@ -43,11 +43,11 @@ foreman run java -cp build/libs/vonage-java-code-snippets-with-dependencies.jar
 If you set the environment variable `QUICKSTART_DEBUG` to any value, extra information
 will be output to the console from the Vonage Server SDK.
 
-## 🙏 Request an Example
+## :octocat: Request an Example
```

Comment:
Can you change the :octocat: to a 🛎️  then I you will be all set to merge.

Reply:
done